### PR TITLE
Ensure consistency in architecture template descriptions

### LINKED ISSUE #446: Ensure consistency in all template descriptions
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

The descriptions in our templates are not consistent with regard to the inclusion of the language. All templates should be updated to include language in the description.

### PR DESCRIPTION
**Pull request by scottslowe**
This PR ensures that each group of architecture templates has a consistent description, and that all descriptions include both the language (C#, Go, Python, TypeScript, or Pulumi YAML) and the platform (AWS, Azure, Google Cloud, or Kubernetes).

Fixes #446

### CODE REVIEW

File: vm-aws-go/Pulumi.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: ${PROJECT}
 description: ${DESCRIPTION}
 runtime: go
 template:
-  description: A virtual machine on Amazon EC2
+  description: A Go program to deploy an instance on AWS EC2
```
Comment (cnunciato) [inline vm-aws-go/Pulumi.yaml:5]:
```suggestion
  description: A Go program to deploy a virtual machine on Amazon EC2
```

File: vm-aws-python/Pulumi.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: ${PROJECT}
 description: ${DESCRIPTION}
 runtime: python
 template:
-  description: A virtual machine on Amazon EC2
+  description: A Python program to deploy an instance on AWS EC2
```
Comment (cnunciato) [inline vm-aws-python/Pulumi.yaml:5]:
```suggestion
  description: A Python program to deploy a virtual machine on Amazon EC2
```

File: vm-aws-typescript/Pulumi.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: ${PROJECT}
 description: ${DESCRIPTION}
 runtime: nodejs
 template:
-  description: A virtual machine on Amazon EC2
+  description: A TypeScript program to deploy an instance on AWS EC2
```
Comment (cnunciato) [inline vm-aws-typescript/Pulumi.yaml:5]:
```suggestion
  description: A TypeScript program to deploy a virtual machine on Amazon EC2
```

File: vm-aws-yaml/Pulumi.yaml [hunk 4]
```diff
@@ -3,7 +3,7 @@ description: ${DESCRIPTION}
 runtime: yaml
 
 template: 
-    description: A virtual machine on Amazon EC2
+    description: A Pulumi YAML program to deploy an instance on AWS EC2
```
Comment (cnunciato) [inline vm-aws-yaml/Pulumi.yaml:6]:
```suggestion
    description: A Pulumi YAML program to deploy a virtual machine on Amazon EC2
```

**Review by cnunciato — APPROVED** [review]
Beautiful. Thanks for doing this! Only a few suggestions, otherwise 🚀 .

**Comment by scottslowe** [comment]
@cnunciato I've committed all your suggestions and enabled auto-merge (squash). Now we just wait for the tests! Thanks for the thorough code review.

File: kubernetes-gcp-yaml/Pulumi.yaml [hunk 5]
```diff
@@ -2,7 +2,7 @@ name: ${PROJECT}
 description: ${DESCRIPTION}
 runtime: yaml
 template:
-  description: A YAML program to create a Kubernetes cluster on Google Cloud
+  description: A YAML program to deploy a Kubernetes cluster on Google Cloud
```
Comment (lukehoban) [inline kubernetes-gcp-yaml/Pulumi.yaml:5]: It looks like everywhere else we changed to `Pulumi YAML`.

```suggestion
  description: A Pulumi YAML program to deploy a Kubernetes cluster on Google Cloud
```

**Review by lukehoban — APPROVED** [review]
LGTM!

**Pull request merged**